src/current/_includes/v23.1/cdc/work-distribution-setting.md

```diff
@@ -0,0 +1 @@
+{% include_cached new-in.html version="v23.1.23" %} You can enable the `changefeed.random_replica_selection.enabled` cluster setting to change the way in which a changefeed distributes work across the cluster. With `changefeed.random_replica_selection.enabled` set to `true`, the job will evenly distribute changefeed work across the cluster by assigning it to any [replica]({% link {{ page.version.version }}/architecture/replication-layer.md %}) for a particular range.
```

Comment:
Worth noting in v23.1 and v23.2 that the changefeed will need to be paused/resumed in order for the changefeed to be replanned after enabling the cluster setting. In v24.1+ the changefeed will be restarted as part of the upgrade process, so there's no need to replan it in those versions.